test/built-ins/Function/prototype/toString/async-generator-method.js

```diff
@@ -0,0 +1,14 @@
+// Copyright 2017 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: pending
+description: Function.prototype.toString on an async generator method
+features: [async-iteration]
+---*/
+
+let f = { /* before */async /* a */ * /* b */ f /* c */ ( /* d */ ) /* e */ { /* f */ }/* after */ }.f;
+let g = { /* before */async /* a */ * /* b */ [ /* c */ "g" /* d */ ] /* e */ ( /* f */ ) /* g */ { /* h */ }/* after */ }.g;
```

Comment:
while this wrapped form is nice, it would be also nice to add a covered name with an expression to be evaluated.

example:

```js
let x = 'y';
let y = { /* before */async /* a */ * /* b */ [ /* c */ x /* d */ ] /* e */ ( /* f */ ) /* g */ { /* h */ }/* after */ }.y;
```